apps/api/src/prisma/schema.prisma

```diff
@@ -518,3 +518,13 @@ model ChangeNotificationSocketMap {
 
   @@index([environmentId, socketId])
 }
+
+model UserEmailChange {
+  id            String    @id @default(cuid())
+  userId        String    
+  newEmail      String
+  otp           String
+  createdOn     DateTime  @default(now())
+
+  @@unique([userId])
```

Comment:
This would be userId + otp, since at a time only one otp can exist for a single user.

Reply:
Made userId+otp unique